app/views/incomes/EditIncomeIrregularHoursView.scala.html

```diff
@@ -37,48 +37,60 @@
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = viewModel.heading,
         preHeadingText = viewModel.preHeading,
-        formForErrorSummary=Some(incomeUpdateForm)
+        formForErrorSummary=Some(incomeUpdateForm),
+        showGdsVersion = true,
     )
 }
 
-@mainTemplate(
+@main(
+    pageTitle = Some(Messages("tai.currentYearSummary.heading")),
     title = messages("tai.irregular.heading"),
-    pageHeader = Some(header),
-    articleClasses = Some("")
+    backLinkContent = Some(Messages("tai.back-link.upper")),
 ){
-    <h2 class="heading-medium">@Html(messages("tai.irregular.introduction", <span class="carry-over">{viewModel.employerName}</span>))</h2>
-    <p>@messages("tai.irregular.introduction.p1", TaxYearRangeUtil.currentTaxYearRange)</p>
+    <div class = "govuk-grid-row">
+        <div class = "govuk-grid-column-two-thirds">
+            @if(incomeUpdateForm.errors.nonEmpty) {
+                @govukErrorSummary(ErrorSummary(errorList = incomeUpdateForm.errors.asTextErrorLinks, title = Text(messages("tai.income.error.form.summary"))))
+            }
 
-    @includes.editIncomeWhatShouldYouInclude()
+            @header
 
-    <p>@messages("tai.irregular.instruction.wholePounds")</p>
+            <h2 class="govuk-heading-m">@Html(messages("tai.irregular.introduction", <span class="carry-over">{viewModel.employerName}</span>))</h2>
+            <p class="govuk-body">@messages("tai.irregular.introduction.p1", TaxYearRangeUtil.currentTaxYearRange)</p>
 
-    @form(action=controllers.income.estimatedPay.update.routes.IncomeUpdateIrregularHoursController.handleIncomeIrregularHours(viewModel.employmentId)) {
-        <div class="form-field-group @if(incomeUpdateForm.hasErrors) {form-group-error}">
-            @incomeUpdateForm.errors(incomeUpdateForm("income").name).map{ error =>
-                <span class="error-notification error-message">@messages(error.message)</span>
+            @includes.editIncomeWhatShouldYouInclude()
 
-            }
+            <p></p>
```

Comment:
accessibility team will pick up on empty p tags  :)  ie produces  unnecessary "noise" for screen readers